frontend/.changeset/warm-suits-love.md

```diff
@@ -0,0 +1,6 @@
+---
+"@liam-hq/erd-core": patch
+"@liam-hq/cli": patch
+---
+
+fix: Update css module for edge marker
```

Comment:
I thought it was refactoring because there was no user impact. what do you think?

```suggestion
:recycle: Update css module for edge marker
```

Reply:
Thanks for good comment!